src/test/ui/borrowck/borrow-raw-address-of-mutability.stderr

```diff
@@ -21,7 +21,7 @@ error[E0596]: cannot borrow `f` as mutable, as it is not declared as mutable
 LL |     let f = || {
    |         - help: consider changing this to be mutable: `mut f`
 LL |         let y = &raw mut x;
-   |                          - calling `f` requires mutable binding due to mutable borrow of `x`
+   |                          - calling `f` requires mutable access due to mutable borrow of `x`
 LL |     };
 LL |     f();
    |     ^ cannot borrow as mutable
```

Comment:
I meant that we could change this message to explain that `f` is not `mut`, but now I see that there's already a `help` to that regard.

So... yea, I don't think this makes anything clearer, but since the old message was confusing to you, we need to make something clearer... Maybe open an issue explaining your reasoning and ping wg-diagnostics?

Reply:
> So... yea, I don't think this makes anything clearer, but since the old message was confusing to you, we need to make something clearer... Maybe open an issue explaining your reasoning and ping wg-diagnostics?

I think I suggested this change because I thought there could be cases where this error was emitted but the solution was not to make the binding mutable. However, it seems the other cases trigger different errors, e.g.:

```rust
fn foo(x: &impl FnMut()) {
    x();
}
```

```
error[E0596]: cannot borrow `*x` as mutable, as it is behind a `&` reference
 --> src/lib.rs:2:5
  |
1 | fn foo(x: &impl FnMut()) {
  |           ------------- help: consider changing this to be a mutable reference: `&mut impl FnMut()`
2 |     x();
  |     ^ `x` is a `&` reference, so the data it refers to cannot be borrowed as mutable
```

So unless there are cases where this error is emitted but the solution is not to make the binding mutable, this PR should probably be closed.